eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState/Types.hs

```diff
@@ -247,50 +254,50 @@ instance Default (IncrementalStake c) where
 --   Given (UTxOState utxo _ _ _ istake) it must be the case that
 --   istake == (updateStakeDistribution (UTxO Map.empty) (UTxO Map.empty) utxo)
 --   Of course computing the RHS of the above equality can be very expensive, so we only
---   use this route in the testing function smartUTxO. But we are very carefull, wherever
+--   use this route in the testing function smartUTxO. But we are very careful, wherever
 --   we update the UTxO, we carefully make INCREMENTAL changes to istake to maintain
 --   this invariant. This happens in the UTxO rule.
 data UTxOState era = UTxOState
-  { utxosUtxo :: !(UTxO era)
-  , utxosDeposited :: !Coin
-  , utxosFees :: !Coin
-  , utxosPpups :: !(State (EraRule "PPUP" era))
-  , utxosStakeDistr :: !(IncrementalStake (EraCrypto era))
+  { sutxosUtxo :: !(UTxO era)
```

Comment:
Oh right, I tried another approach earlier with type families, so `ShelleyLedgerState` became the concrete type instance, but since I'm not going with that any longer, it makes sense to revert it back to just `LedgerState`.